src/commands/build/build-command-utils.ts

```diff
@@ -110,6 +111,22 @@ export async function build({
       entryPoints: [libEntry, ...componentEntries]
     });
   });
+
+  // Generates Custom Elements Manifest file.
+  if (!config.context.customElementsManifestConfig?.disabled) {
+    await runTask('Generating custom elements manifest...', async () => {
+      let cmd = 'npx custom-elements-manifest analyze';
+
+      if (config.context.customElementsManifestConfig?.configFileName) {
+        cmd += `--config ${config.context.customElementsManifestConfig.configFileName}`;
+      } else {
+        cmd += ` --globs "**/*.ts"`;
+      }
+
+
+      return await runCommand(cmd, stagingSrcDir, false);
```

Comment:
I might suggest we move this to its own async function just to clean it up a little instead of having this inline. It will also make it easier for us to call it if we do add a separate command for generating the manifest on its own.